### PR TITLE
Add Ukrainian and Russian translation.

### PR DESCRIPTION
**Pull request by vlad36N**
Add Ukrainian and Russian translation.

### CODE REVIEW

File: custom_components/places/translations/ua.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                    "name": "Назва датчика",
+                    "devicetracker_id": "DeviceTracker Ідентифікатор",
+                    "home_zone": "Домашня зона",
+                    "options": "Параметри відображення",
+                    "map_provider": "Постачальник карт",
+                    "map_zoom": "Масштаб карти",
+                    "api_key": "Електронна пошта для використання як ключа OpenStreetMap API (необов’язково)",
+                    "language": "Мова (необов'язково)",
+                    "extended_attr": "Увімкнути розширені атрибути",
+                    "show_time": "Показати час останнього оновлення в кінці стану "(з xx:yy)"",
```
Comment (Snuffy2) [inline custom_components/places/translations/ua.json:17]: There should be single quotes around the parenthesis

File: custom_components/places/translations/ua.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                "data": {
+                    "devicetracker_id": "DeviceTracker Ідентифікатор",
+                    "home_zone": "Домашня зона",
+                    "options": "Параметри відображення",
+                    "map_provider": "Постачальник карт",
+                    "map_zoom": "Масштаб карти",
+                    "api_key": "Електронна пошта для використання як ключа OpenStreetMap API (необов’язково)",
+                    "language": "Мова (необов'язково)",
+                    "extended_attr": "Увімкнути розширені атрибути",
+                    "show_time": "Показати час останнього оновлення в кінці стану "(з xx:yy)"",
```
Comment (Snuffy2) [inline custom_components/places/translations/ua.json:37]: There should be single quotes around the parenthesis

File: custom_components/places/translations/ua.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                    "use_gps_accuracy": "Використовуйте точність GPS"
+                },
+                "description": "**Оновлення датчика:&nbsp;{sensor_name}**\nДетальніше див. [Параметри конфігурації]({component_config_url}) на GitHub"
+            }
+        }
+    },
+    "issues": {
+        "deprecated_yaml": {
+            "title": "Видаліть конфігурацію YAML Places",
+            "description": "Вашу наявну конфігурацію YAML було автоматично імпортовано в інтерфейс користувача.\in\Видаліть конфігурацію YAML Місця зі свого файлу configuration.yaml і перезапустіть Home Assistant, щоб вирішити цю проблему."
```
Comment (Snuffy2) [inline custom_components/places/translations/ua.json:47]: The JSON linter doesn't like the two \ in the string. Not sure if this is actually correct or not though.

File: custom_components/places/translations/ru.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                    "use_gps_accuracy": "Используйте точность GPS"
+                },
+                "description": "**Обновление сенсора:&nbsp;{sensor_name}**\nПодробности см. в [Параметры конфигурации]({component_config_url}) на GitHub."
+            }
+        }
+    },
+    "issues": {
+        "deprecated_yaml": {
+            "title": "Удалить конфигурацию Places YAML",
+            "description": "Ваша существующая конфигурация YAML была автоматически импортирована в пользовательский интерфейс.\in\Удалите конфигурацию Places YAML из файла configuration.yaml и перезапустите Home Assistant, чтобы решить эту проблему."
```
Comment (Snuffy2) [inline custom_components/places/translations/ru.json:47]: The JSON linter doesn't like the two \ in the string. Not sure if this is actually correct or not though.

**Review by Snuffy2 — CHANGES_REQUESTED** [review]
Thanks! I do not speak either of these languages so I'll rely on you for their accuracy. There appear to be a couple of small formatting errors.

**Comment by vlad36N** [comment]
Fixed. Sorry my bad.

File: custom_components/places/translations/ru.json [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+                    "use_gps_accuracy": "Используйте точность GPS"
+                },
+                "description": "**Обновление сенсора:&nbsp;{sensor_name}**\nПодробности см. в [Параметры конфигурации]({component_config_url}) на GitHub."
+            }
+        }
+    },
+    "issues": {
+        "deprecated_yaml": {
+            "title": "Удалить конфигурацию Places YAML",
+            "description": "Ваша существующая конфигурация YAML была автоматически импортирована в пользовательский интерфейс.\n\n\Удалите конфигурацию Places YAML из файла configuration.yaml и перезапустите Home Assistant, чтобы решить эту проблему."
```
Comment (Snuffy2) [inline custom_components/places/translations/ru.json:47]: Is the \У correct? It is making it appear as invalid JSON.

**Review by Snuffy2 — CHANGES_REQUESTED** [review]

File: custom_components/places/translations/ru.json (hunk 5 above)
Comment (vlad36N) [reply custom_components/places/translations/ru.json]: Sorry

**Review by Snuffy2 — APPROVED** [review]

**Pull request merged**